it/mail/chrome/messenger/preferences/preferences.properties

```diff
@@ -1,49 +1,49 @@
 #### Add HTML and Plain Text Domain Names for sendOption
 
-html_domainsAddDomainTitle=Aggiungere nome dominio HTML:
+html_domainsAddDomainTitle=Aggiungi nome dominio HTML:
 html_domainsAddDomain=Nome dominio HTML:
-plaintext_domainsAddDomainTitle=Aggiungere un nome dominio testo semplice:
+plaintext_domainsAddDomainTitle=Aggiungi nome dominio testo semplice:
 plaintext_domainsAddDomain=Nome dominio testo semplice:
 
-domainNameErrorTitle=Errore nel nome di dominio
+domainNameErrorTitle=Errore nome di dominio
 #### LOCALIZATION NOTE: do not translate %S
 domainDuplicationError=Il nome di dominio %S esiste già in una lista HTML o Testo semplice.
 
 #### Junk
-confirmResetJunkTrainingTitle=Confermare
-confirmResetJunkTrainingText=Si è certi di voler azzerare i dati di autoistruzione del filtro incrementale?
+confirmResetJunkTrainingTitle=Conferma
+confirmResetJunkTrainingText=Sei sicuro di voler ripristinare i dati di training del filtro adattivo?
 
 #### Master Password
 
-setMasterPassword=Set Master Password...
-changeMasterPassword=Cambiare la master password ...
+setMasterPassword=Imposta Password principale...
+changeMasterPassword=Cambia la Password principale...
 
 #### Downloads
-desktopFolderName=Desktop
+desktopFolderName=Scrivania
 myDownloadsFolderName=Download
-chooseAttachmentsFolderTitle=Choose Folder
+chooseAttachmentsFolderTitle=Scegli cartella
 
 #### Applications
 
-fileEnding=file %S
-saveFile=Save File
+fileEnding=File %S
+saveFile=Salva file
 
 # LOCALIZATION NOTE (useApp, useDefault): %S = Application name
-useApp=Use %S
-useDefault=Use %S (default)
+useApp=Utilizza %S
+useDefault=Utilizza %S (predefinito)
 
-useOtherApp=Use other...
-fpTitleChooseApp=Selezionare l'applicazione di appoggio
-manageApp=Application Details...
-alwaysAsk=Always ask
-delete=Delete Action
-confirmDeleteTitle=Delete Action
-confirmDeleteText=Are you sure you want to delete this action?
+useOtherApp=Utilizza altri...
+fpTitleChooseApp=Seleziona l'applicazione di appoggio
+manageApp=Informazioni applicazione...
+alwaysAsk=Chiedi sempre
+delete=Elimina azione
+confirmDeleteTitle=Elimina azione
+confirmDeleteText=Sei sicuro di voler eliminare questa azione?
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+confirmDeleteText=Sei sicuro di voler eliminare questa Azione?